anymail/backends/mailpace.py

```diff
@@ -0,0 +1,190 @@
+            status_msg = "error"
+            id = None
+
+        if status_msg == "queued":
+            # Add the message_id to all of the recipients
+            for recip in payload.to_cc_and_bcc_emails:
+                recipient_status[recip.addr_spec] = AnymailRecipientStatus(
+                    message_id=id, status="queued"
+                )
+        elif status_msg == "error":
+            if "errors" in json_response:
+                for field in ["to", "cc", "bcc"]:
+                    if field in json_response["errors"]:
+                        error_messages = json_response["errors"][field]
+                        for email in payload.to_cc_and_bcc_emails:
+                            for error_message in error_messages:
+                                if (
+                                    "undefined field" in error_message
+                                    or "is invalid" in error_message
+                                ):
+                                    recipient_status[
+                                        email.addr_spec
+                                    ] = AnymailRecipientStatus(
+                                        message_id=None, status="invalid"
+                                    )
+                                elif "contains a blocked address" in error_message:
+                                    recipient_status[
+                                        email.addr_spec
+                                    ] = AnymailRecipientStatus(
+                                        message_id=None, status="rejected"
+                                    )
+                                elif (
+                                    "number of email addresses exceeds maximum volume"
+                                    in error_message
+                                ):
+                                    recipient_status[
+                                        email.addr_spec
+                                    ] = AnymailRecipientStatus(
+                                        message_id=None, status="invalid"
+                                    )
+                        else:
+                            continue  # No errors found in this field; continue to next field
+            else:
+                raise AnymailRequestsAPIError(
+                    email_message=message,
+                    payload=payload,
+                    response=response,
+                    backend=self,
+                )
+
+        return dict(recipient_status)
+
+
+class MailPacePayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+        }
+        self.server_token = backend.server_token  # esp_extra can override
+        self.to_cc_and_bcc_emails = []
+        self.merge_data = None
+        self.merge_metadata = None
```

Comment:
(There's no need to have `merge_data` and `merge_metadata` properties if MailPace doesn't support batch merge.)